### PR TITLE
Tools reference edit

### PR DESCRIPTION
**Pull request by suearkinunity**
### Purpose of this PR

Update the Tools reference pages:

* Edit Cut tool, PolyShape tool, and Shape tool pages to reflect new reality.
* Add Snap to Grid page.

Didn't need to update the options in the individual shape pages, so I didn't touch those at all.

### Links

**Jira:** DOCATT-5850

### Comments to Reviewers

Loads of changes here. Read very, very carefully.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=538) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=538) before we can accept your contribution.<br/><hr/>**Sue Arkin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=538) it.</sub>

File: Documentation~/TableOfContents.md [hunk 1]
```diff
@@ -100,9 +100,8 @@
 	* [Triangulate - Objects](Object_Triangulate)
 	* [Weld Vertices](Vert_Weld)
 * [Tools reference](ref_tools)
-	* [Cut tool](cut-tool)
-	* [Poly Shape tool](polyshape)
-	* [Shape tool](shape-tool)
+	* [New PolyShape and Edit PolyShape tools](polyshape)
+	* [New Shape and Edit Shape tools](shape-tool)
```
Comment (JonMUnity) [inline Documentation~/TableOfContents.md:104]: I think "New" at the beginning of these entries is just setting ourselves up for more maintenance work in the future and, potentially, user confusion. We could always just link to them in the What's New page to indicate that they're newer.

File: Documentation~/polyshape.md [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
+The **New PolyShape** tool stays active until you deactivate it, so you can create many shapes in a sequence. 
 
->  **Note**: If you are in editing mode immediately after creating a new Poly Shape Mesh, you can also click **New Poly Shape** (![Poly Shape Icon](images/icons/NewPolyShape.png)) on the ProBuilder toolbar to exit the Poly Shape tool.
+> **Tip:** You can add shapes next to each other, but they stay separate objects. To merge them, use the [Merge Objects](Object_Merge.md) tool.
 
+## Edit a PolyShape
 
+To edit an existing PolyShape:
 
+1. Select the PolyShape in the **Scene** view.
```
Comment (JonMUnity) [inline Documentation~/polyshape.md:27]:
```suggestion
1. Select the PolyShape in the Scene view.
```

File: Documentation~/ref_tools.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+| --- | --- |
+| [New Shape and Edit Shape](shape-tool.md) | Creates a new mesh with the Shape component, which defines the mesh's shape primitive. |
+| [New PolyShape and Edit PolyShape](polyshape.md) | Creates a new mesh with the PolyShape component. You then draw a 2D shape, and extrude it to a 3D shape. |
+| [Cut](cut-tool.md) | Creates a subface in an existing mesh face. You design the shape of the new face by defining points on the mesh, then move the face as you do any other face. <br /><br />You can use this tool on any face, regardless of whether you created the mesh with the [New Shape](shape-tool.md) tool, the [New PolyShape](polyshape.md) tool, or by [probuilderizing](Object_ProBuilderize.md) a standard Unity mesh. |
+
+> **Note:** For documentation on the [Bezier Shape](bezier.md) tool, refer to [Experimental features](experimental.md).
+
+## Active context in the Tools overlay
+
+The ProBuilder tools are available in the **Tools** overlay in the **Scene** view. The tools available depend on the active context: [[link to context]]
```
Comment (JonMUnity) [inline Documentation~/ref_tools.md:19]:
```suggestion
The ProBuilder tools are available in the **Tools** overlay in the Scene view. The tools available depend on the active context: [[link to context]]
```

File: Documentation~/ref_tools.md [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 
+* You can create and edit shapes and PolyShapes from both the ProBuilder and GameObject contexts.
+* The Cut tool is available only when the context is ProBuilder.
 
-> **Note:** For documentation on the [Bezier Shape](bezier.md) tool, refer to the [Experimental features](experimental.md) section of the documentation.
+**Tip:** While a tool is active, performing any other task - a task that doesn't use the tool - deactivates the tool. For example, if while using the [Cut](cut-tool.md) tool on an object you click on a different object, the tool exits without performing the cut on the original object.
```
Comment (JonMUnity) [inline Documentation~/ref_tools.md:24]:
```suggestion
**Tip:** While a tool is active, performing any other task that doesn't use the tool deactivates the tool. For example, if while using the [Cut](cut-tool.md) tool on an object, you click on a different object, the tool exits without performing the cut on the original object. 
```

File: Documentation~/shape-tool.md [hunk 5]
```diff
@@ -1,63 +1,44 @@
...
-![When you create a new shape, the Create Shape properties appear. When you edit an existing shape, the Edit Shape properties appear.](images/shape-tool.png)
+To create a new shape:
 
-**(A)** Shape icons let you choose the shape's primitive when you are drawing a new shape.
+1. Initiate the New Shape tool in one of the following ways:
 
-**(B)** The **Tools** panel displays the Edit Shape icon ![Edit Shape icon](images/icons/Panel_Shapes.png) when you have a Shape Mesh selected. You can toggle this icon to turn the editing mode on and off.
+    * In the **Tools** overlay, click the **New Shape** icon. A short click creates the selected shape; a long click opens a selection menu to choose a different shape.
+    * In the main menu, go to **Tools** > **ProBuilder** > **Editors** > **New Shape** and select a shape.
+1. Draw points in the **Scene** view to create the 2D outline of the shape. To help you draw in an exact location, use [**Snap to Grid**](snap-to-grid.md).
```
Comment (JonMUnity) [inline Documentation~/shape-tool.md:13]:
```suggestion
1. Draw points in the Scene view to create the 2D outline of the shape. To help you draw in an exact location, use [**Snap to Grid**](snap-to-grid.md).
```

File: Documentation~/shape-tool.md [hunk 6]
```diff
@@ -1,63 +1,44 @@
...
+1. When you connect the last point to the first point, you can extrude the shape to create a 3D object.
 
-**(C)** The **Snapping** option only appears when you use ProBuilder inside a version of Unity prior to 2021.1. It enables auto-snapping for the Shape tool. For more information, see [Snapping](#Snapping).
+## Edit a shape
 
-**(D)** The **Shape Properties** section lets you switch to a new shape primitive and define the size of the bounding box. When you are drawing a new shape, the **Pivot** property lets you specify whether to place the pivot at the corner or in the middle of the bounding box. For details, see [Choosing the shape and size](#ShapeProperties).
+To edit an existing shape:
 
-**(E)** The **&lt;Shape&gt; settings** section lets you customize the new shape with properties that are specific to each shape. For example, you can change the number of sides (faces) for both cylinders and cones, but for cylinders you can also set the smoothing group and set segment heights for the sides. For details, see [Customizing specific shapes](#ShapeSpecificSettings).
+1. Select the shape in the **Scene** view.
```
Comment (JonMUnity) [inline Documentation~/shape-tool.md:20]:
```suggestion
1. Select the shape in the Scene view.
```

File: Documentation~/shape-tool.md [hunk 7]
```diff
@@ -1,63 +1,44 @@
...
+1. In the **Tools** overlay, set the active context to ProBuilder. 
+1. In the **Tools** overlay, click **Edit ProBuilder Shape** to:
+    * Open the **Shape Settings** window. Refer to [Shape settings](#shape-settings)
+    * Activate an element selection (face, edge, or vertex) mode and redesign the shape. 
 
+## Shape settings
 
+Each shape has its own settings. To access the settings for a shape, select the shape in the **Scene** view and click **Edit Shape** in the **Tools** overlay to open the **Shape Settings** window. These settings are also available in the **Inspector** window.
```
Comment (JonMUnity) [inline Documentation~/shape-tool.md:28]:
```suggestion
Each shape has its own settings. To access the settings for a shape, select the shape in the Scene view and click **Edit Shape** in the **Tools** overlay to open the **Shape Settings** window. These settings are also available in the **Inspector** window.
```

File: Documentation~/snap-to-grid.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Snap to grid
```
Comment (JonMUnity) [inline Documentation~/snap-to-grid.md:1]: The content on this page duplicates the information in the Grid and Snap manual page. Do you think it's worth repeating here?

File: Documentation~/TableOfContents.md (hunk 1 above)
Comment (suearkinunity) [reply Documentation~/TableOfContents.md:104]: It's new v edit, not new v old

File: Documentation~/ref_tools.md (hunk 3 above)
Comment (suearkinunity) [reply Documentation~/ref_tools.md:19]: Actually hang on. 
Write "Scene" in bold with a capital S, followed by "view" in lowercase.
That's from the style guide.

File: Documentation~/snap-to-grid.md (hunk 8 above)
Comment (suearkinunity) [reply Documentation~/snap-to-grid.md:1]: Well it was already in the ProBuilder docs so I assumed it should stay. It's also way shorter and in context, which might be nicer for people.

File: Documentation~/ref_tools.md (hunk 3 above)
Comment (suearkinunity) [reply Documentation~/ref_tools.md:19]:
```suggestion
The ProBuilder tools are available in the **Tools** overlay in the **Scene** view. The tools available depend on the active context: GameObject or ProBuilder.
```

Comment (suearkinunity) [reply Documentation~/ref_tools.md:19]: Can't link to it - it hasn't been published yet.

Comment (JonMUnity) [reply Documentation~/ref_tools.md:19]: > Actually hang on.
Write "Scene" in bold with a capital S, followed by "view" in lowercase.
That's from the style guide.

When did we change this?!

Comment (JonMUnity) [reply Documentation~/ref_tools.md:19]: Please disregard my suggestions for that then.

File: Documentation~/snap-to-grid.md (hunk 8 above)
Comment (suearkinunity) [reply Documentation~/snap-to-grid.md:1]: I'll leave it in, then

File: Documentation~/polyshape.md [hunk 9]
```diff
@@ -1,43 +1,44 @@
...
+The **New PolyShape** tool stays active until you deactivate it, so you can create many shapes in a sequence. 
 
->  **Note**: If you are in editing mode immediately after creating a new Poly Shape Mesh, you can also click **New Poly Shape** (![Poly Shape Icon](images/icons/NewPolyShape.png)) on the ProBuilder toolbar to exit the Poly Shape tool.
+> **Tip:** You can add shapes next to each other, but they stay separate objects. To merge them, use the [Merge Objects](Object_Merge.md) tool.
 
+## Edit a PolyShape
 
+To edit an existing PolyShape:
 
+1. Select the PolyShape in the Scene view.
```
Comment (JonMUnity) [inline Documentation~/polyshape.md:27]:
```suggestion
1. Select the PolyShape in the **Scene** view.
```

**Review by JonMUnity — APPROVED** [review]

**Comment by suearkinunity** [comment]
@modrimkus please can you merge this?

**Pull request merged**